### ISSUE TITLE
Playlist imports

### ISSUE THREAD

**Issue by alejocd** [HemantKArya/BloomeeTunes#74]
Hi, wanted to congratulate you on the app, I use It every day. However I am having issues with playlist imports from Spotify links. Whenever I try that, the importing stops randomly at different number of songs, not importing the whole playlist . It also seems to be necessary to have the screen turned on for the duration of that process.

**Comment by HemantKArya** [comment]
Thank you so much @alejocd for your appreciation, it really means a lot to me. This issue is most probably going to be solved in next released especially about the not all songs are importing there is reason behind that cause the algorithm tries to find the most related song available on YouTube platform, so if song exist but with bit different name then most probably it fails to recognise it as the right one. But still it shouldn't fail for next ones so it will be fixed in next release.

**Comment by alejocd** [comment]
Thank you, will be waiting for the update

**Comment by HemantKArya** [comment]
Hi @alejocd this issue is now resolved in the latest version if there is any song which is failed to import then it automatically moves to next songs to fetch and import. Can you try and let me know here?

**Comment by alejocd** [comment]
Hi, it seems to have been solved. However, not all songs were transferred, is it because the program does not find the right song in YT?

**Comment by HemantKArya** [comment]
Yes, right now, it searches the most similar title and artist name so that we can add the correct song for the songs available in the playlist.
